_component/countdown-timer/content/_documentation.md

```diff
@@ -0,0 +1,15 @@
+<h3>Installation</h3>
+
+<p>This component accepts two arguments, the selector for the countdown timer container, and an optional settings object containing callback functions and other options.</p>
+
+<div class="u-spacing__bottom--medium">
+<code>npm install @10up/countdown-timer --save</code>
+</div>
+
+<h3>API and Options</h3>
+
+<p><a href="https://github.com/10up/component-countdown-timer#api">Refer to the full API documentation</a> for details on available callback functions and settings.</p>
+
+<h3>A Note on Time Zones</h3>
+
+When creating a countdown timer, keep in mind that the remaining time shown by the component will depend on the end user's local time zone. You can control the behavior by providing time zone information in your markup. <a href="https://github.com/10up/component-countdown-timer#dealing-with-time-zones">Refer to the full documentation</a> for more details.
```

Comment:
Let's make baseline the main point of documentation so we're not relying on the readme file for links like this